packages/@aws-cdk/aws-lambda/test/test.lambda.ts

```diff
@@ -1525,6 +1526,184 @@ export = {
 
     test.done();
   },
+
+  'default function with CDK created Profiling Group'(test: Test) {
```

Comment:
Move these tests into test.function.ts and use the new assertion style to assert specific resources instead of matching the template - https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-lambda/test/test.function.ts#L258